### PR TITLE
DOC: All integer values must be non-negative

### PR DESCRIPTION
**Pull request by LeonaTaric**
issue#22745 

```
cdef class SeedSequence():
        note: all integer values must be non-negative.

cdef class BitGenerator():
        note: all integer values must be non-negative.
```

### CODE REVIEW

**Comment by charris** [comment]
@bashtage  Comment?

File: numpy/random/bit_generator.pyx [hunk 1]
```diff
@@ -260,6 +260,7 @@ cdef class SeedSequence():
     ----------
     entropy : {None, int, sequence[int]}, optional
         The entropy for creating a `SeedSequence`.
+        note: all entropy not None must be non-negative.
```
Comment (bashtage) [inline numpy/random/bit_generator.pyx:263]: It isn't common to use `note`.  Just something along of the lines "All integer values must be non-negative."  Same below.

**Comment by charris** [comment]
@LeonaTaric Could you make the suggested changes concerning "note:"?

**Comment by LeonaTaric** [comment]
> ll entropy not None must be non-negative.

Yeah，I'll change "note" to "comment" when I have time

**Comment by charris** [comment]
The suggestion was  "All integer values must be non-negative", neither "comment" nor "note" is needed to introduce the information.

**Comment by LeonaTaric** [comment]
> The suggestion was "All integer values must be non-negative", neither "comment" nor "note" is needed to introduce the information.

aha

**Comment by LeonaTaric** [comment]
image

Emmmmm, what's up, anyone who can tell me how to recheck the cancelled checks

**Comment by LeonaTaric** [comment]
> > The suggestion was "All integer values must be non-negative", neither "comment" nor "note" is needed to introduce the information.
> 
> aha



> image
> 
> Emmmmm, what's up, anyone who can tell me how to recheck the cancelled checks

@charris excuse me, do you know? I don't know what should I do to solve this

**Comment by bashtage** [comment]
You probably want to push the rided doc string first.  When you push a new commit, the CI checks will run again.

**Comment by LeonaTaric** [comment]
> You probably want to push the fixed doc string first. When you push a new commit, the CI checks will run again.

so I need to commit an any commit, even it does nothing, only one empty line. tks.

**Comment by bashtage** [comment]
The easiest way is to amend your previous commit and then force push.  Or just commit an empty (`--allow-empty`) and then push that.  This said, you still need to fix the doc string, so do that and then commit and push.

**Comment by LeonaTaric** [comment]
> The easiest way is to amend your previous commit and then force push. Or just commit an empty (`--allow-empty`) and then push that. This said, you still need to fix the doc string, so do that and then commit and push.

Thanks, I close and reopen this PR can do it too!

**Comment by LeonaTaric** [comment]
finally, all success

**Comment by LeonaTaric** [comment]
@charris emmmmmm, 3 weeks, can I be merged

**Comment by charris** [comment]
@LeonaTaric  I check this regularly and you haven't yet implemented the suggestions.

**Comment by LeonaTaric** [comment]
> @LeonaTaric I check this regularly and you haven't yet implemented the suggestions.

However, I think I have yet implemented the suggestions, because all the notes have been "all integer values must be non-negative."

**Comment by LeonaTaric** [comment]
Oh, I forget to change the title and content of this PR, I only change the commit.
Now the title and content has been edited.

**Comment by LeonaTaric** [comment]
> The suggestion that you haven't implemented was to remove the `note:`.

Oh No, I will change it at night (in my timezone)

**Comment by LeonaTaric** [comment]
@charris Now I have implemented the suggestion successfully that I have removed the "note:"

**Review by bashtage — APPROVED** [review]
LGTM.

**Comment by charris** [comment]
Thanks @LeonaTaric .

**Pull request merged**